README.md

```diff
@@ -1,66 +1,66 @@
-# Hedera Smart Contracts
+## Foundry
```

Comment:
Revert these README changes.
You accidentally removed the valid test.

I'd advise you move this to a `FOUNDRY_TESTING.MD` file and then add a section in this README uner `## Development guidelines` pointing to it.

Reply:
Which "valid test" did I accidentially remove?

Reply:
Mistyped test, meant README section content